internal/oci/runtime_oci.go

```diff
@@ -481,7 +481,23 @@ func (r *runtimeOCI) ExecContainer(ctx context.Context, c *Container, cmd []stri
 	args := r.defaultRuntimeArgs()
 	args = append(args, "exec", "--process", processFile, c.ID())
 
-	execCmd := cmdrunner.CommandContext(ctx, c.RuntimePathForPlatform(r), args...) //nolint: gosec
+	var execCmd *exec.Cmd
+	if execCgroupPath := c.ExecCgroupPath(); execCgroupPath != "" {
```

Comment:
this looks to me it would always do the exec cgroup path, even if the container is not in a high perf hook. am I missing something?